### PR TITLE
Adjusted Naming Schema of Calico Roles

### PR DESCRIPTION
**Pull request by VincentS**
Adjusted Naming of roles to be able to see which role is currently executed and renamed two template files to have the same name as the final files on the hosts to be consistent in naming.

### CODE REVIEW

**Comment by ingvagabund** [comment]
aos-ci-test

File: roles/calico/tasks/main.yml [hunk 1]
```diff
@@ -7,20 +7,20 @@
...
+- name: Calico Node | Assure the calico certs have been generated
   stat:
     path: "{{ item }}"
   with_items:
   - "{{ calico_etcd_ca_cert_file }}"
   - "{{ calico_etcd_cert_file}}"
   - "{{ calico_etcd_key_file }}"
 
-- name: Configure Calico service unit file
+- name:  Calico Node | Configure Calico service unit file
```
Comment (ingvagabund) [inline roles/calico/tasks/main.yml]: Please, remove one space before `Calico Node prefix`. The same for the remaining 9 occurrences.

**Comment by VincentS** [comment]
Made changes based on feedback from @ingvagabund

File: roles/calico/tasks/main.yml [hunk 2]
```diff
@@ -29,46 +29,46 @@
     enabled: yes
   register: start_result
 
-- name: Assure CNI conf dir exists
+- name: Calico Node | Assure CNI conf dir exists
   become: yes
   file: path="{{ cni_conf_dir }}" state=directory
 
-- name: Generate Calico CNI config
+- name:  Calico Node | Generate Calico CNI config
```
Comment (ingvagabund) [inline roles/calico/tasks/main.yml]: ^ here as well

Comment (ingvagabund) [inline roles/calico/tasks/main.yml]: Once done, please squash the commits

Comment (VincentS) [inline roles/calico/tasks/main.yml]: @ingvagabund Done :)

**Comment by VincentS** [comment]
PR modified based on feedback; Commits squashed and rebased

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
[merge]

**Pull request merged**